tests/parser/WellTests.cpp

```diff
@@ -1954,15 +2031,28 @@ END
     const auto es    = EclipseState { deck };
     const auto sched = Schedule { deck, es };
 
-    const auto& wellP = sched.getWell("P", 0);
-    BOOST_CHECK_EQUAL(wellP.getProductionProperties().BHPTarget.get<double>(), 5.0);
-
-    const auto& wellI = sched.getWell("I", 0);
-    BOOST_CHECK_CLOSE(wellI.getInjectionProperties().BHPTarget.get<double>(), 30.0, 1e-12);
```

Comment:
This is a little unexpected.  Did these accidentally get dropped from the [`FBHPDEF_Basic`](https://github.com/OPM/opm-common/blob/9f3fdea7df1f72de552c34142207193d615dfa65/tests/parser/WellTests.cpp#L1882) test above?

Reply:
good catch. yeah, i definitely mangled the diff here somewhat. remedied.